indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java

```diff
@@ -0,0 +1,19 @@
+package au.org.aodn.esindexer.model;
+
+import lombok.Getter;
+
+@Getter
+public enum MediaType {
+    TEXT_HTML("text/html"),
+    TEXT_XML("text/xml"),
+    IMAGE_JPEG("image/jpeg"),
```

Comment:
where is this one being used?

Reply:
IMAGE_JPEG

Reply:
They are from good examples of stac collection. For now, not in use. Maybe they should be added later when they start using?